src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/ElectricJuicer.java

```diff
@@ -0,0 +1,28 @@
+package io.github.thebusybiscuit.slimefun4.implementation.items.electric.machines;
+
+import io.github.thebusybiscuit.slimefun4.api.items.ItemGroup;
+import io.github.thebusybiscuit.slimefun4.api.items.SlimefunItemStack;
+import io.github.thebusybiscuit.slimefun4.api.recipes.RecipeType;
+import io.github.thebusybiscuit.slimefun4.core.attributes.RecipeDisplayItem;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.abstractItems.AContainer;
+import org.bukkit.Material;
+import org.bukkit.inventory.ItemStack;
+
+import javax.annotation.Nonnull;
+
+public class ElectricJuicer extends AContainer implements RecipeDisplayItem {
```

Comment:
Can you please add javadocs to this class?

Reply:
Sure thing